CMakeLists.txt

```diff
@@ -481,11 +481,10 @@ ELSE()
 ENDIF(WIN32)
 
 #this is the product (a library)
-if(${build_as_dynamic})
-    add_library(aziotsharedutil SHARED ./src/aziotsharedutil.def ${source_c_files} ${source_h_files})
-else()
-    add_library(aziotsharedutil ${source_c_files} ${source_h_files})
-endif()
+add_library(aziotsharedutil ${source_c_files} ${source_h_files})
+
+#make sure we can link as dll/so
+add_library(aziotsharedutil_dll SHARED ./src/aziotsharedutil.def ${source_c_files} ${source_h_files})
 
```

Comment:
Actually I am hesitant to do that.
I'd like to understand first if this dll is to be used or it should be built only as validation of the fact that it can be built.
If it is to be used, yes, I can add it to the install targets.

---
In reply to: [118512431](https://github.com/Azure/azure-c-shared-utility/pull/78#discussion_r118512431) [](ancestors = 118512431)